bin/inx-chronicle/src/metrics.rs

```diff
@@ -28,7 +29,7 @@ impl MetricsWorker {
 
 pub struct MetricsState {
     server_handle: (JoinHandle<()>, Option<oneshot::Sender<()>>),
-    process_metrics_handle: JoinHandle<()>,
+    abort_handles: Vec<AbortHandle>,
```

Comment:
Don't actually need to store these if you don't want to. You can just use `cx.abort().await` in the parent actor's shutdown fn.

Reply:
So I don't actually need to return them in the `spawn_task` method?

Reply:
It is not strictly necessary, but I would do it anyway because it could be useful.